build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+fun properties(key: String) = project.findProperty(key).toString()
+
+plugins {
+    // Java support
+    id("java")
+    // Kotlin support
+    id("org.jetbrains.kotlin.jvm") version "1.5.10"
+    // gradle-intellij-plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
+    id("org.jetbrains.intellij") version "1.0"
```

Comment:
I haven't upgraded it here to the latest version yet but it's still an upgrade to v1.0. 

And there are some deprecations in v1.1.4 which I wanna address first.

Reply:
@stukalin any pointers on what the next steps are?

- update the plugin to the latest version
- update the IntelliJ version
- release the plugin

like that? 😅